ens-improvement-proposals/ensip-XX-platform-source-parameter.md

```diff
@@ -0,0 +1,57 @@
+
+---
+**Authors**: @slobo.eth and Avsa.eth
+
+---
+**Status**: draft
+
+---
```

Comment:
This needs a 'description' header and the authors and status should be in a table. Please see other ENSIPs for reference to the format.